Userland/Games/2048/Game.h

```diff
@@ -119,6 +119,7 @@ class Game final {
     u32 m_target_tile { 0 };
 
     bool m_evil_ai { false };
+    bool m_want_to_continue { false };
```

Comment:
I still don't see this being reset to `false` anywhere.

Reply:
@alimpfard look at `start_a_new_game` when it is called, it automatically create new game object, which default construct it with false? is not it ?

Reply:
Yep, sorry, my bad.

Reply:
That's fine. :)